data/mods/ssb/conditions.ts

```diff
@@ -866,10 +866,22 @@ export const Conditions: {[k: string]: ModdedConditionData} = {
 			this.add(`c|${getName('KingSwordYT')}|Mucho texto`);
 		},
 		onSwitchOut() {
-			this.add(`c|${getName('KingSwordYT')}|Hasta la próximaaaa`);
+			this.add(`c|${getName('KingSwordYT')}| /teleport`);
```

Comment:
This seems like a mistake

Reply:
yes it is
ive commited and pushed, this prepush thingy is taking its sweet time